webull.py

```diff
@@ -596,6 +643,17 @@ def get_bars(self, stock=None, interval='m1', count=1, extendTrading=0) :
             df.loc[datetime.fromtimestamp(int(row[0]))] = data
         return df.iloc[::-1]
 
+
+    def get_calendar(self):
+        params = {'type': 'm1', 'count': 1, 'extendTrading': 0}
```

Comment:
no comment, no headers, and hardcoding 'aapl'? is the goal just to see what the most recent market date was?

Reply:
Yes, I've added a header now, and let the symbol needed be passed in..  but it seems to be the only way to get market open/close times and make a guess on if it's a if its a federal holiday